tests/robustness/validate/watch.go

```diff
@@ -117,8 +118,8 @@ func validateReliable(t *testing.T, events []model.WatchEvent, report report.Cli
 		}
 		for _, resp := range op.Responses {
 			for _, event := range resp.Events {
-				if events[index].Match(op.Request) && events[index] != event {
-					t.Errorf("Broke watch guarantee: Reliable - a sequence of events will never drop any subsequence of events; if there are events ordered in time as a < b < c, then if the watch receives events a and c, it is guaranteed to receive b, event missing: %+v", events[index])
+				if events[index].Match(op.Request) && !events[index].Equals(event) {
```

Comment:
```suggestion
				if events[index].Match(op.Request) && events[index].PersistedEvent != event.PersistedEvent {
```

We don't want/need t validate whole WatchEvent, just the peristedEvent within it.